next.config.js

```diff
@@ -1,22 +1,37 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+// defaults to local
+let gqlAddr = 'http://localhost:8080/query';
+const GUAC_GQL_ADDR = process.env.GUAC_GQL_ADDR;
+
+try {
+  let fileContents = fs.readFileSync(GUAC_GQL_ADDR, 'utf8');
```

Comment:
from user perspective, I would have set the value to the gql endpoint instead of the file path to `guac.yaml`. maybe use another name?

Reply:
@tuananh that makes sense. check my updated changes to see if it aligns to your suggestion